### DISCUSSION TITLE
WinSCP screenshots

### DISCUSSION THREAD

**Discussion by doberbauer** [mcguirepr89/BirdNET-Pi#846] [General]
Hi all,
I wasn't sure where to direct this so I apologize if this is the wrong place.
In [Backup and Restore the Database](https://github.com/mcguirepr89/BirdNET-Pi/wiki/Backup-and-Restore-the-Database) under Windows Users there's a note about needing screenshots from WinSCP. I'd be happy to provide those.

**Comment by gerdesj** [comment]
@ehpersonal38 - https://github.com/mcguirepr89/BirdNET-Pi/wiki/Backup-and-Restore-the-Database 

"(Sorry, no screenshots yet -- I don't have access to a Windows machine. If you do, and are willing to provide screenshots, please let me know and I will be grateful to add them here for other users."

**Comment by ehpersonal38** [comment]
Sure, if you'd like to take screenshots that would be helpful!

**Comment by doberbauer** [comment]
WinSCP will open with a login prompt. Enter the hostname of your pi, its username and password. and click Login. 
![Screenshot 2023-04-11 111747](https://user-images.githubusercontent.com/81889558/231604507-93980026-5127-4c68-80ae-f7d7872f0828.png)

If it's your first time connecting to the host it'll present a warning. Click yes. 
![Screenshot 2023-04-11 111829](https://user-images.githubusercontent.com/81889558/231605199-e1044410-b5a5-4e78-ac6e-c3c1753ef149.png)

The left side is your PC's directory and the right is your Pi's. On the Pi side, open the BirdNET-Pi directory
![Screenshot 2023-04-11 111918](https://user-images.githubusercontent.com/81889558/231605952-6aef76fc-4888-4c87-831e-dc80276b8c60.png)

Open the scripts directory and find birds.db
![Screenshot 2023-04-11 112039](https://user-images.githubusercontent.com/81889558/231606074-614616c9-6d74-4ba5-aa80-57f1443cfee3.png)

Click and drag or copy/paste birds.db to your PC's directory.

![Screenshot 2023-04-11 112140](https://user-images.githubusercontent.com/81889558/231606386-d5e7eb9e-b2c5-4fcc-88cc-41fc98290269.png)
![Untitled1](https://user-images.githubusercontent.com/81889558/231606457-1a41a1e3-093e-4739-bbcf-4b1e03d7a28c.png)

**Comment by ehpersonal38** [comment]
That's brilliant - thank you so much! Just one request - the screenshots are actually for restoring the database, not backing it up. So if you're able to re-do the last 2 screenshots I'll get this up on the Wiki 😄

**Comment by doberbauer** [comment]
Ah my bad. 
![image](https://user-images.githubusercontent.com/81889558/231610562-029894d3-b165-4776-a38b-d316fae0a9d6.png)

![image](https://user-images.githubusercontent.com/81889558/231610773-cafc0ecb-aa6a-4325-9efc-d1b43836efc9.png)
